pkg/mbedtls/include/riot_mbedtls_config.h

```diff
@@ -229,6 +229,487 @@ extern "C" {
 #endif
 /** @endcond */
 
+/**
+ *
+ * @brief Enable the generic SSL/TLS code.
+ *
+ * Module:  library/ssl_tls.c
+ * Caller:  library/ssl_cli.c
+ *          library/ssl_srv.c
+ *
+ * Requires: MBEDTLS_CIPHER_C, MBEDTLS_MD_C
+ *           and at least one of the MBEDTLS_SSL_PROTO_XXX defines
+ *
+ * This module is required for SSL/TLS.
+ */
+#if CONFIG_MBEDTLS_SSL_TLS_C
```

Comment:
The `CONFIG_*` symbols are expected to be generated by Kconfig, but they are not yet included to the Kconfig configuration system. The [here](https://github.com/RIOT-OS/RIOT/blob/master/pkg/mbedtls/contrib/entropy/Kconfig) and [here](https://github.com/RIOT-OS/RIOT/blob/master/pkg/mbedtls/contrib/entropy/Kconfig) for reference.